saleor/graphql/app/mutations/app_create.py

```diff
@@ -76,7 +76,9 @@ def perform_mutation(cls, _root, info, /, **data):
 
     @classmethod
     def save(cls, info, instance, cleaned_input):
-        instance.identifier = graphene.Node.to_global_id("App", instance.pk)
         instance.save()
+        if not instance.identifier:
+            instance.identifier = graphene.Node.to_global_id("App", instance.pk)
```

Comment:
Where does the identifier come in this mutation, if not generated from ID? It's not available in the input.

Reply:
Good point. 
Context: it is added to the input in related PR (but only to the newest version)

I will remove that unnecessary if.